DABI.js

```diff
@@ -0,0 +1,321 @@
+{
+    "translatorID": "5cf8bb21-e350-444f-b9b4-f46d9fab7827",
+    "label": "DABI",
+    "creator": "Jens Mittelbach",
+    "target": "^https?://dabi\\.ib\\.hu-berlin\\.de/",
+    "minVersion": "1.0",
+    "maxVersion": "",
+    "priority": 100,
+    "inRepository": true,
+    "translatorType": 4,
+    "browserSupport": "gcsibv",
+    "lastUpdated": "2016-01-08 11:43:21"
+}
+
+function detectWeb(doc, url) {
+    if (doc.title.trim().indexOf("DABI. Datensatz Vollanzeige") == 0) {
+        return "journalArticle";
+    } else if (doc.title.trim().indexOf("DABI: Rechercheergebnis") == 0) {
+        var keinTreffer = doc.getElementsByTagName("br")[7].nextSibling.data.indexOf("Keine Treffer für die Suche nach:")
+        if (keinTreffer == -1) {
+            return "multiple";
+        }
+    }
+}
+
+function doWeb(doc, url) {
+    var ids = [];
+
+    //If Statement checks if page is a Search Result, then saves requested Items
+    if (detectWeb(doc, url) == "multiple") {
+        Z.selectItems(getSearchResults(doc), function(items) {
+            if (!items) return true;
+            for (var i in items) {
+                ids.push(i);
+            }
+        });
+    } else if (detectWeb(doc, url) == "journalArticle") {
+        //saves single page items
+        ids = [url];
+    }
+    ZU.processDocuments(ids, scrape);
+}
+
+
+function getSearchResults(doc) {
+    var trs = doc.getElementsByTagName("tr"),
+        tds = null,
+        items = {};
+
+    for (var i = 1; i < trs.length; i++) {
+        tds = trs[i].getElementsByTagName("td");
+        for (var n = 0; n < tds.length; n++) {
+            var url = doc.location.origin + "/cgi-bin/dabi/" + tds[0].firstChild.getAttribute("href"),
+                author = tds[1].innerHTML,
+                title = tds[2].innerHTML.replace(/<br>/g, '. ');
+
+            if (!author == '') {
```

Comment:
This says `(!author) == ''`, which works, but is quite odd. `if (!author)` would be better.

Reply:
Done